cmd/brew-test-bot.rb

```diff
@@ -978,7 +1022,10 @@ def cleanup_shared
 
       Tap.names.each do |tap|
         next if tap == "homebrew/core"
+        next if tap == "homebrew/test-bot"
         next if tap.end_with?("/test-bot")
```

Comment:
I had to fix a conflict here but wasn't sure; does it look like OK @davidchall?

Reply:
L1025 and L1026 are both preventing the `test-bot` repo from being untapped. In 1d684ea6b8eac6800e63c9d89e66e6dba528d254 I replaced `next if tap == "homebrew/test-bot"` with `next if tap.end_with?("/test-bot")` so that my forked repo was also spared from this untapping. So we don't really need to add this line back in – the case is already covered by the following line.

Reply:
Thanks! I just updated to reflect that (I removed `next if tap == "homebrew/test-bot"`)